### DISCUSSION TITLE
please read

### DISCUSSION THREAD

**Discussion by travelstartup26** [community/community#113852] [Codespaces]
### Select Topic Area

Question

### Body

im looking for an app developers to get on board my startup. I need someone who believes in the idea, myself, and the company. were based in nyc. once you sign an NDA form that I provide we can talk more about the details. the industry is travel. nobody is going to outwork me so I expect the same from my future partner/(s) I am giving up equity in my dream in return for someone who is ready to disrupt an industry and change the world.

**Comment by queenofcorgis** [comment]
Hi @travelstartup26 , we're happy you're here 😀

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
